source/_posts/2019-03-11-upgrading-to-embark-4.md

```diff
@@ -28,7 +32,7 @@ Converting to a third party pipeline is easy. This can be done with three simple
 
 NOTE: If you are planning on using Embark's built-in Webpack pipeline (and not use a third party pipeline), please [skip down to the remainder of the Embark 4 breaking changes](#New-Web3-plugin).
 
-Embark 4 generates contract artifacts for all of the contracts in your Dapp. These artifacts enable importing the Dapp's contracts in to the Dapp's javascript. Most of these artifacts were already generated before, but lived inside the `.embark/` folder. Since most modern frontend build systems require source files to live inside of a very specific source folder, we have given developers the opportunity to specify the destination folder for these artifacts, allowing the frontend build tool to pack them in to the build files.
+Embark 4 generates [Smart Contract artifacts](/docs/javascript_usage.html#Embark-Artifacts) for all of the Smart Contract in your Dapp. These artifacts enable importing the Dapp's Smart Contracts into the Dapp's source code. Most of these artifacts were already generated before, but lived inside the `.embark/` folder. Since most modern frontend build systems require source files to live inside of a very specific source folder, we have given developers the opportunity to specify the destination folder for these artifacts, allowing the frontend build tool to pick them up for processing.
```

Comment:
Actually only really changed the word `contracts` here to `Smart Contracts`.

I did the same throughout documentation by the way. While `contracts` is convenient, it doesn't really mean anything or a hundred things. 

I think we should always be explicit with this terminology in written form for clarity.